### DISCUSSION TITLE
Jumpstart template sets default URL that doesn't exist

### DISCUSSION THREAD

**Discussion by indirect** [avo-hq/avo#1914] [Help] [answered]
I used the Jumpstart Rails template to install Avo:

```
rails app:template LOCATION=https://v3.avohq.io/templates/jumpstart-pro.template
```

(This template does not appear to be on GitHub anywhere, so I am opening a discussion instead of an issue on that repo.)

The generated `config/initializers/avo.rb` file includes these two lines:

```ruby
  config.root_path = "/avo"
  config.home_path = "/dashboards/overview"
```

Unfortunately, these two lines do not work together--going to `/avo` produces a redirect to `/dashboards/overview`, which does not exist.

I would suggest checking which version of Avo is being installed, and for community redirecting to `nil`, and for pro or advanced redirecting to `/avo/dashboards/overview`.

**Comment by adrianthedev** [comment, marked as answer]
Thanks @indirect for all the support. 
The template is located here https://github.com/avo-hq/jumpstart-pro-app-template and [here](https://v3.avohq.io/templates/jumpstart-pro).
I've added a comment in the template so folks know where to find it.

Yup! Top feedback! Updated the repo!
